spec/file-icons-spec.coffee

```diff
@@ -21,3 +25,45 @@ describe 'FileIcons', ->
     FileIcons.resetService()
 
     expect(FileIcons.getService()).not.toBe(service)
+
+  
+  describe 'Class handling', ->
```

Comment:
Okay, moving it out of the top-level `describe` seemed to do the trick... no idea why, but at least it's in the one file now, as per your request.